docs/form-customization.md

```diff
@@ -357,6 +357,18 @@ const uiSchema = {
 };
 ```
 
+#### `copyable` option
+
+A copy button is shown by default for an item if `items` contains a schema object, or the item is an `additionalItems` instance. You can turn this off with the `copyable` option in `uiSchema`:
```

Comment:
Can we hide the copy button by default? This way existing arrays will look the same.

Reply:
Yes we can, but I need to update the playground, test, and the docs again.
The reason I set it to true is to make the default state same as the others, I have update the playground, test, and docs for that.

Reply:
Yeah, but adding a new button to arrays is quite a large change and definitely a breaking change. I think we can add it in but only if it's hidden by default.